fblldb.py

```diff
@@ -41,6 +41,7 @@ def loadCommandsInDirectory(commandsDirectory):
 def loadCommand(module, command, directory, filename, extension):
   func = makeRunCommand(command, os.path.join(directory, filename + extension))
   name = command.name()
+  helpText = command.description().split('\n', 1)[0] # first line of description
```

Comment:
That's definitely more readable. The use of `maxsplit` was to avoid doing more work than necessary.